controllers/team_controller.py

```diff
@@ -13,7 +13,7 @@
 
 
 class TeamList(CacheableHandler):
-    VALID_PAGES = [1, 2, 3, 4, 5, 6, 7, 8]
+    VALID_PAGES = range(1, (8000 // 1000) + 2)
```

Comment:
This approach feels a little confusing from several magic numbers. If we add a comment that gives a little information (`support up to 8999 teams - 1000 per page`) that would probably be helpful. Better yet - we could make these numbers constants and give the constants some comments and descriptive names.

Also changing to `(9000 // 1000) + 1` and having a comment about making the range inclusive in Python feels slightly better than the `+ 2` (they both kinda blow but - one feels better IMO)

Reply:
Yeah, not sure what I like here, but I added in some constants and got rid of the +2 bit